gokart/gcs_config.py

```diff
@@ -9,4 +9,4 @@ class GCSConfig(luigi.Config):
 
     def get_gcs_client(self) -> luigi.contrib.gcs.GCSClient:
         return luigi.contrib.gcs.GCSClient(
-            oauth_credentials=os.environ.get(self.gcs_credential_name))
+            oauth_credentials=os.getenv(self.gcs_credential_name, self.gcs_credential_name))
```

Comment:
Default value is the same as value, is this correct?

Reply:
@hirosassa Yes, it is.
- token='hogehoge' (directory)  :  hogehoge
- token=${TOKEN}  :  TOKEN from os.getenv
- token=%(TOKEN)s  :  TOKEN from os.getenv
- token='hogehoge' (directory but not found)  :  exception
- token=%(TOKEN)s (not found)  :  exception  
- token=${TOKEN (not found) }  :  exception

Reply:
Understood! cool!